src/modules/partition/gui/ChoicePage.cpp

```diff
@@ -1575,7 +1576,9 @@ ChoicePage::calculateNextEnabled() const
         }
     }
 
-    if ( m_config->installChoice() != InstallChoice::Manual && m_encryptWidget->isVisible() )
+    if ( m_config->installChoice() != InstallChoice::Manual 
+         && (m_encryptWidget->isVisible() ||
+                    m_encryptWidget->isEncryptionCheckboxChecked()))
```

Comment:
This seems wrong to me.  If the encryption widget isn't visible, encryption should be disabled.

Reply:
First time we go on this condition, we are at opening of the application.
Page is not visible so widget is not visible.
This is not a problem when Encryption is not checked (default behaviour), Next button will be accessible.
But if Encryption is pre Check, Next button will still be accessible, even if we have no
Encryption passphrase set. 
We can't let user to go to Next page until the passphrase is set or Encryption Widget is not check.

Only way I found to have the right behaviour in all combinations is to verify Encryption Checkbox at this point.

Reply:
OK, I tested this and the behavior seems fine.  Although, it is a little aggressive about rechecking that checkbox.